Add services overview page (#4681)
Qiskit/documentation: docs/guides/compute-services.mdx

Qiskit Runtime is designed to use additional classical and quantum compute resources, including techniques such as error suppression and error mitigation, to return a higher-quality result from executing quantum circuits on quantum processors. Examples include dynamical decoupling for error suppression, and readout mitigation and zero-noise extrapolation (ZNE) for error mitigation. Learn how to configure these options on the [Configure error mitigation](./configure-error-mitigation) page.

Qiskit Runtime also includes three types of execution modes for running your quantum program on IBM hardware: [_Job_, _Session_, and _Batch_,](/docs/guides/execution-modes) each of which have different use cases and implications for the quantum job queue. A Job is a single query to a primitive that can be run over a specified number of shots. Sessions allow you to efficiently run multiple jobs in iterative workloads on quantum computers. Batch mode allows you to submit all your jobs at once for parallel processing. Note that Open Plan users cannot submit session jobs.

To quickly install Qiskit Runtime, run the following command:
```bash
pip install qiskit-ibm-runtime
```

> More details on setting up a development environment for building quantum programs can be found in the [installation page](./install-qiskit).

Reviewer: ```suggestion
See the [installation page](/docs/guides/install-qiskit) for details on setting up a development environment for building quantum programs.
```
